src/overlays/actors/ovl_En_Rz/z_en_rz.c

```diff
@@ -314,12 +311,14 @@ s32 func_80BFBCEC(EnRz* this, PlayState* play) {
         (player->stateFlags2 & PLAYER_STATE2_2000000)) {
         return true;
     }
-
     return false;
 }
 
+/*
+* Updates either Link's or Rosa Sisters animations every frame
```

Comment:
This is confusing, it sounds like it might be the Player's own animation instead of a LinkAnimation as used by this actor. It's fine to omit the comment, it's a short enough function that it's easy to tell what it's doing anyway.